### PR TITLE
Prep for Release - storage/stg95 blob&files preview

### PR DESCRIPTION
**Pull request by tanyasethi-msft**
- [x] The purpose of this PR is explained in this or a referenced issue.
- [x] The PR does not update generated files.
   - These files are managed by the codegen framework at [Azure/autorest.go][].
- [x] Tests are included and/or updated for code changes.
- [x] Updates to module CHANGELOG.md are included.
- [x] MIT license headers are included in each file.

[Azure/autorest.go]: https://github.com/Azure/autorest.go

### CODE REVIEW

File: sdk/storage/azblob/internal/exported/version.go [hunk 1]
```diff
@@ -8,5 +8,5 @@ package exported
 
 const (
 	ModuleName    = "github.com/Azure/azure-sdk-for-go/sdk/storage/azblob"
-	ModuleVersion = "v1.4.2"
+	ModuleVersion = "v1.5.1-beta.1"
```
Comment (jhendrixMSFT) [inline sdk/storage/azblob/internal/exported/version.go:11]: This should be `v1.5.0-beta.1`.

File: sdk/storage/azfile/internal/exported/version.go [hunk 2]
```diff
@@ -8,5 +8,5 @@ package exported
 
 const (
 	ModuleName    = "github.com/Azure/azure-sdk-for-go/sdk/storage/azfile"
-	ModuleVersion = "v1.3.2"
+	ModuleVersion = "v1.4.1-beta.1"
```
[the remark this answers is not in the document]
Comment (tanyasethi-msft) [reply sdk/storage/azfile/internal/exported/version.go]: Changed, thanks!

File: sdk/storage/azblob/internal/exported/version.go (hunk 1 above)
Comment (tanyasethi-msft) [reply sdk/storage/azblob/internal/exported/version.go]: Changed, thanks!

**Review by jhendrixMSFT — APPROVED** [review]

**Comment by tanyasethi-msft** [comment]
Closing, since the '&' in the branch name could not be parsed by the pipelines. This PR is re created with the changed branch name

**Comment by jhendrixMSFT** [comment]
@benbp is the `&` in a branch name known to be problematic?

**Comment by benbp** [comment]
@jhendrixMSFT this will be fixed by @praveenkuttappan, we're just missing some quotes in the yaml templates.

**Pull request closed without merging**